proposals/NNNN-swiftpm-extensible-build-tools.md

```diff
@@ -240,6 +240,8 @@ protocol TargetBuildContext {
     /// That path of the package that contains the target.
     let packageDir: Path
     
+    // [Tom] should this be a function instead of a property to potentially save performance?
```

Comment:
Maybe.  But it could also be a computed property that returns a lazy collection.  The strings do need to be sent down to the extension, but can presumably be done in a way that doesn't have to send complete paths (e.g. just root directory path and then subpaths.  Seemed to me that as a computed property we can do what we'd want to implement this API.

I should actually add all the `{get}` accessors and make these `let`s be `var`s.